example_analysis/4.run_sbs_phenotype_processes.sh

```diff
@@ -4,7 +4,16 @@
 BRIEFLOW_PATH="../"
 CONFIG_FILE_PATH="config/config.yml"
 
-# Run only the preprocess rules
+# TODO: Run both processes when done with testing
+
+# Run the SBS process rules
+snakemake --use-conda --cores all \
+    --snakefile "${BRIEFLOW_PATH}workflow/Snakefile" \
+    --configfile "$CONFIG_FILE_PATH" \
+    --until all_sbs_process
+
+# Run the phenotype process rules
 snakemake --use-conda --cores all \
     --snakefile "${BRIEFLOW_PATH}workflow/Snakefile" \
-    --configfile "$CONFIG_FILE_PATH" \
```

Comment:
Please, can you ensure that all the processes.sh files always have an until flag? Even if it is the last module you have worked on? For the future.

Reply:
Yep! Sorry I missed this